Multisig/Data/Services/Notification Service/UnregisterNotificationTokenRequest.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UnregisterNotificationTokenRequest.swift
+//  Multisig
+//
+//  Created by Moaaz on 8/5/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct UnregisterNotificationTokenRequest: JSONRequest {
+    let deviceID: UUID
```

Comment:
this rather should be string, or we need to test that the UUID formatting is the same as the backend expects it

Reply:
ah, no, let's see, no change needed